crates/bitwarden/src/crypto/enc_string.rs

```diff
@@ -170,6 +172,29 @@ impl EncString {
         s.map(|s| s.parse()).transpose()
     }
 
+    /// TODO: Convert this to a trait method
```

Comment:
We probably want to rework all this logic and ensure we differentiate asymmetric and symmetric enc strings. And expand our key locator, decryption pattern to handle assymetric keys.